debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/tools/ddrinteractive/commands/VersionInfoCommand.java

```diff
@@ -0,0 +1,73 @@
+/*******************************************************************************
+ * Copyright (c) 2020, 2020 IBM Corp. and others
+ *
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/
+ * or the Apache License, Version 2.0 which accompanies this distribution and
+ * is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * This Source Code may also be made available under the following
+ * Secondary Licenses when the conditions for such availability set
+ * forth in the Eclipse Public License, v. 2.0 are satisfied: GNU
+ * General Public License, version 2 with the GNU Classpath
+ * Exception [1] and GNU General Public License, version 2 with the
+ * OpenJDK Assembly Exception [2].
+ *
+ * [1] https://www.gnu.org/software/classpath/license.html
+ * [2] http://openjdk.java.net/legal/assembly-exception.html
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+ *******************************************************************************/
+package com.ibm.j9ddr.vm29.tools.ddrinteractive.commands;
+
+import java.io.PrintStream;
+import java.util.Properties;
+
+import com.ibm.j9ddr.CorruptDataException;
+import com.ibm.j9ddr.tools.ddrinteractive.Command;
+import com.ibm.j9ddr.tools.ddrinteractive.Context;
+import com.ibm.j9ddr.tools.ddrinteractive.DDRInteractiveCommandException;
+import com.ibm.j9ddr.tools.ddrinteractive.annotations.DebugExtension;
+import com.ibm.j9ddr.vm29.pointer.helper.J9JavaVMHelper;
+import com.ibm.j9ddr.vm29.view.dtfj.DTFJContext;
+
+public class VersionInfoCommand extends Command{
+    
+    public VersionInfoCommand() {
+		addCommand("versioninfo", "", "Prints out the java version of the JDK");
+	}
+	
+	public void run(String command, String[] args, Context context, PrintStream out)
+			throws DDRInteractiveCommandException 
+	{
+		initialize();
+		Object javaFullVersion = properties.get("java.fullversion");
+		Object javaRuntimeVersion = properties.get("java.runtime.version");
+		Object javaRuntimeName = properties.get("java.runtime.name");
+		if (javaRuntimeName == null) {
```

Comment:
When I tried this, both `javaRuntimeName` and `javaRuntimeVersion` were `null`: we need to understand why those system properties are not available.

Reply:
> When I tried this, both `javaRuntimeName` and `javaRuntimeVersion` were `null`: we need to understand why those system properties are not available.

I am unsure why this is the case. I tested on a couple of different builds for both IBM and OpenJ9 and I saw the expected output